api/Dockerfile

```diff
@@ -101,7 +101,6 @@ COPY --chown=opener . /api/
 # Collect static assets, these are used by the next stage, `nginx`
 RUN env \
   SETUP_ES="False" \
-  STATIC_ROOT="/static" \
```

Comment:
This is now correctly configured in settings for local and production deployments (which are intercepted by Nginx anyway) so there's no need to repeat the configuration here.